yoyodyne/models/lstm.py

```diff
@@ -339,8 +339,8 @@ def beam_decode(
                 char_likelihoods,
                 decoder_hiddens,
             ) in likelihoods:
-                # -> B x seq_len x outputs.
-                # This is 1 x 1 x outputs since we fixed batch size to 1.
+                # -> B x seq_len x output_size.
```

Comment:
Maybe just remove this comment.
The one below (sans "this is") should be sufficient.